tests/unit/core/server/testServer.js

```diff
@@ -157,20 +157,35 @@ describe(__filename, () => {
         .set('cookie', `${defaultConfig.get('cookieName')}="${token}"`)
         .end();
 
-      const { api, user } = store.getState();
+      const {
+        api,
+        user,
+        reduxAsyncConnect: reduxAsyncConnectState,
+      } = store.getState();
 
       expect(response.statusCode).toEqual(200);
       expect(api.token).toEqual(token);
       expect(user.id).toEqual(42);
       expect(user.username).toEqual('babar');
       mockUserApi.verify();
+
+      const { loadState } = reduxAsyncConnectState;
+      expect(loadState).toEqual({});
```

Comment:
I guess this is a safety measure since you're relying on it in the next test? I'd rather not rely on this state because it will be removed in https://github.com/mozilla/addons-frontend/issues/2356

Reply:
This can be removed here but the check is quite useful to be sure the rendering was successful.